CONTRIBUTING.md

```diff
@@ -300,20 +300,19 @@ Typeshed includes `scripts/create_baseline_stubs.py`.
 It generates stubs automatically using a tool called
 [stubgen](https://mypy.readthedocs.io/en/latest/stubgen.html) that comes with mypy.
 
-To get started, fork typeshed, clone your fork, and then
-[create a virtualenv](#-or-create-a-local-development-environment).
-You can then install the library with `pip` into the virtualenv and run the script below,
+To get started, fork typeshed, clone your fork, and then [prepare your environment](#preparing-the-environment).
+You can then install the library with `uv pip` into the virtualenv and run the script below,
 replacing `$INSERT_LIBRARY_NAME_HERE` with the name of the library:
 
 ```bash
-(.venv)$ pip install $INSERT_LIBRARY_NAME_HERE
+(.venv)$ uv pip install $INSERT_LIBRARY_NAME_HERE
```

Comment:
I think contributors shouldn't need to know about `uv`. While I'm fine with using `uv` to speed up our CI, it's not something I would personally use when developing locally, and I don't think it solves a problem for the typical contributor.

Reply:
I _do_ use `uv` all the time locally, but I'd also much prefer it if new contributors don't have to worry about it. I really want new contributors to be able to get up and running as quickly as possible on typeshed; the setup steps should be as simple as possible and involve the bare minimum of custom tools IMO.

Reply:
I've only changed places where I'd assume `pip install -r requirements-tests.txt` was already run.

Using it in doc was less about install speed, and more about not having to worry about virtual environments, and whether it exists / is activated.

I could see being told to run a custom tool (uv) when you may not even know that it was part of the installed requirements to be jarring.
I've updated the PR with that in mind.